docs/src/main/sphinx/release/release-395.md

```diff
@@ -0,0 +1,93 @@
+# Release 395 (7 Sep 2022)
+
+## General
+
+* Reduce memory consumption of fault-tolerant execution. ({issue}`13855`)
+* Reduce memory consumption of aggregations. ({issue}`12512`)
+* Improve performance of aggregations with decimals. ({issue}`13573`)
+* Improve concurrency for large clusters. ({issue}`13934`, `13986`)
+* Remove `information_schema.role_authorization_descriptors` table. ({issue}`11341`)
+* Fix inaccurate results when using `SHOW CREATE TABLE` or `SHOW COLUMNS` on a
+  table with columns that use a reserved keyword as column name. ({issue}`13483`)
+
+## ClickHouse connector
+
+* Fix query failure when renaming or dropping a column which should be quoted. ({issue}`13839`)
+
+## Delta Lake connector
+
+* Add support for the `ALTER TABLE ... RENAME TO` statement with a Glue
+  metastore. ({issue}`12985`)
+* Significantly improve performance of inserts by automatically scaling the
```

Comment:
Please drop the "Significantly", that is a subjective term and we don't necessarily know how much things will improve, someone could be already using a high task.writer-count in their existing config.
Also, inserts -> writes as the improvement is not specific to INSERT query, CTAS query could benefit too.

Reply:
Agreed

Reply:
Please apply this comment to the hive and iceberg sections as well